packages/css/src/types.ts

```diff
@@ -642,10 +652,15 @@ export interface Theme {
   printColorModeName?: string
 
   /**
-   * Adds styles defined in theme.styles.root to the <body> element along with color and background-color
+   * Adds styles defined in theme.styles.root to the <html> element along with color and background-color
    */
-  useBodyStyles?: boolean
+  useRootStyles?: boolean
```

Comment:
I think this might be a bit confusing - `useRootStyles` is declared as optional boolean, however it is tested as potentially `null` and we also allow for `useBodyStyles`. 

What do you think about having a theme prop 'rootElement?: string` - by default this is 'body' but user can change it to any other tag.
And we keep just `useBodyStyles` (even though confusing since it can be not just body anymore) for compatibility sake?

Reply:
Thanks for the explanation, makes sense, I was just expressing my pov :)

However, we definitely need to keep backward compatibility (as you have it in your implementation) imho.